### PR TITLE
bytes 0.5 compat

### PR DESCRIPTION
**Pull request by Fabian-Gruenbichler**
captured.rs is still broken, see #42

Signed-off-by: Fabian Grünbichler <fabian.gruenbichler@student.tuwien.ac.at>

### CODE REVIEW

**Comment by partim** [comment]
Thanks for the PR!

I’d like to do one more release in the current 0.4 series with #44, and then we can move to 0.5 with bytes 0.5.

File: src/encode/primitive.rs [hunk 1]
```diff
@@ -4,7 +4,7 @@
 //! parent.
 
 use std::io;
-use bytes::{BufMut, Bytes, BytesMut};
+use bytes::{buf::ext::BufMutExt, Bytes, BytesMut};
```
Comment (partim) [inline src/encode/primitive.rs]: Minor nitpicking: Could you move that into a separate line? I.e.,

```
use bytes::buf::ext::BufMutExt;
```

**Comment by partim** [comment]
I created a `CaptureBuilder` to deal with the `Bytes::extend_by_slice` issue. Can you maybe merge master into your branch? We should be good to go then.

**Comment by Fabian-Gruenbichler** [comment]
done (rebased on current master), build and test runs through locally - let's see what CI has to say ;)

**Comment by partim** [comment]
I sneakily moved to Github Actions in master and voila: all good.

Thanks for doing this work!

**Pull request merged**